src/Switches.h

```diff
@@ -95,6 +96,33 @@ class SwitchOnCycleDuration {
 		g_cycleStartTime = nextCycleStartTime;
 	}
 };
+
+//=============================================================================
+class SwitchOnDummySend {
+public:
+	// dummy send between two cycles starting point and take starting point of next cycle
+
+	inline void execute(Message *pMsgRequest, int ifd) {
+		uint8_t buffer[pMsgRequest->getLength()];
```

Comment:
*this line is not C code*, but a gcc extension - hence, you must make sure it compiles on all supported platforms, like Windows and BSD 
if there is no additional example in SockPerf for this trick, I recommend that you will use static array of size MAX_MSG_LEN